database/migrations/platform/2023_10_29_000000_update_player_games_table.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+declare(strict_types=1);
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+return new class() extends Migration {
+    public function up(): void
+    {
+        Schema::table('player_games', function (Blueprint $table) {
+            $table->index(['game_id', 'achievements_unlocked']);
+            $table->index(['game_id', 'achievements_unlocked_hardcore']);
+        });
+    }
+
+    public function down(): void
+    {
+        Schema::table('player_games', function (Blueprint $table) {
+            $table->dropIndex('player_games_game_id_achievements_unlocked_index');
+            $table->dropIndex('player_games_game_id_achievements_unlocked_hardcore_index');
```

Comment:
```php
$table->dropIndex(['game_id', 'achievements_unlocked']);
$table->dropIndex(['game_id', 'achievements_unlocked_hardcore']);
```

Reply:
Thanks, that's much easier indeed. Resolved in latest. I'll keep hunting for more optimizations.